__tests__/app.test.js

```diff
@@ -69,3 +69,25 @@ describe("/api/articles/:article_id", () => {
             });
     });
 });
+
+describe("/api/articles", () => {
+    test("GET:200 sends an array of articles to the client sorted by date in descending order", () => {
+        return request(app)
+            .get("/api/articles")
+            .expect(200)
+            .then(({ body: articles }) => {
+                expect(articles.length).toBe(articleData.length);
```

Comment:
We know how many articles we are expecting back, so we should do an assertion that is static